README.md

```diff
@@ -1 +1,163 @@
-# Data-Science-Exhibition
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+
+# 📖 Capstone Project: Data Science Exhibition 2023 <a name="about-project"></a>
+
+> This project is about AI & Big Data Exhibition that will take place at Riverside Drive on 19th and 20th January 2023. Please visit the website to see details of the project.
+
+[Loom Video Presentation](https://loom.com/share/622d645485cf4c4fb8ce72c97813eedb)
+
+## 🛠 Built With <a name="built-with"></a>
+
+- **JavaScript**
+- **HTML**
+- **CSS**
+
+### Key Features <a name="key-features"></a>
+
+- **Responsiveness**
+- **Interactivity**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
```

Comment:
- [ ] I can't find any live demo link working in your readme. According to the project, the requirement you should use GitHub Pages to deploy your website and make sure that all pages, links, and images work as expected then add the link in the README. Please fix it :+1: